src/com/cloudogu/ces/cesbuildlib/Git.groovy

```diff
@@ -260,4 +329,88 @@ class Git implements Serializable {
             script.sh "git ${args}"
         }
     }
-}
+
+    /**
+     * Finishes a git flow release and pushes all merged branches to remote
+     *
+     * Only execute this function if you are already on a release branch
+     *
+     * @param releaseVersion the version that is going to be released
+     */
+    void finishGitRelease(String releaseVersion) {
```

Comment:
The flow is very opinionated, perhaps this should be in a separate class too.

Reply:
Funnily, I considered implementing release flows in a generic and reusable way a long time ago but gave up: #1.
But I think a `GitFlow` class would be a good option.

Reply:
I moved this to the new GitFlow class.